src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3099,6 +3105,14 @@ def create_vmss(cmd, vmss_name, resource_group_name, image=None,
                                                                 build_msi_role_assignment, build_nsg_resource,
                                                                 build_nat_rule_v2)
 
+    # Breaking Change Warning, change image alias
+    if image:
+        if image == "Win2008R2SP1":
+            logger.warning('[Breaking Change] On April 04, 2023,'
+                           'the image deployed using "Win2008R2SP1" will reach its end of life.')
```

Comment:
Same as above

Reply:
Received, already modified in https://github.com/Azure/azure-cli/pull/25930/commits/6b89f74dede91ad0569b445a5f1d1ad32234b50f